WarehouseManagement.Core/Services/DeliveryService.cs

```diff
@@ -64,7 +63,13 @@ public async Task<DeliveryDto> GetByIdAsync(int id)
                         MarkerId = dm.MarkerId,
                         MarkerName = dm.Marker.Name
                     })
-                    .ToList()
+                    .ToList(),
+                EntriesFinishedProcessing = d
+                    .Entries.Where(e => e.FinishedProccessing.HasValue)
+                    .Count(),
+                EntriesWaitingProcessing = d
+                    .Entries.Where(e => !e.FinishedProccessing.HasValue)
```

Comment:
Maybe like this:
`EntriesWaitingProcessing = d.Entries.Where(e => !e.FinishedProcessing.HasValue && !e.StartedProcessing.HasValue`)
*This is in case you want to retrieve the count of the entries which are waiting (because now you are getting the ones that are waiting and being processed), if the logic is other, please correct me.

Reply:
You are absolutely correct, but I spoke with Kris about it, and he said that we group waiting and processing entries under the waiting status for display to the front end.

Reply:
Ok then, thanks :)